examples/README.md

```diff
@@ -0,0 +1,6 @@
+# Examples
+
+For those of you who are new to Go, check out
+[pkg.go.dev](https://https://pkg.go.dev/github.com/charmbracelet/bubbletea) to
```

Comment:
Should this be `log`?

```suggestion
[pkg.go.dev](https://https://pkg.go.dev/github.com/charmbracelet/log) to
```

Reply:
Thanks for catching this! Yes, it should be